notebooks/crop_area_estimation.ipynb

```diff
@@ -9,20 +9,20 @@
     "\n",
```

Comment:
<p>Is <code>getFeatures</code> function really necessary here? If a function remains it would be clearer to use either features or coords in both the get function and the variable name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>It's really not necessary, I'll remove it </p>